examples/vedo_ex1.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from magicclass import magicclass, field
+from magicclass.ext.vtk import VedoCanvas
+import vedo
+
+@magicclass
+class VedoViewerUI:
+    canvas = field(VedoCanvas)
+
+ui = VedoViewerUI()
+
+coords = np.random.randn(1000, 3)
+data = np.cos(coords[:,1])
+
+# create a points object and a set of axes
+points = vedo.Points(coords)
+points.cmap("viridis", data).add_scalarbar3d()
+axes = vedo.Axes(points, c="white")
+
+# create a histogram of data
+histo = vedo.pyplot.histogram(
+    data, 
+    bins=10, 
+    c="viridis",
+    title="", xtitle="", ytitle="",
+).clone2d("bottom-right", 0.25)
+
```

Comment:
Is it possible to use methods like `ui.canvas.add_points()` here as well (of course, if the method is out-of-date, I'll take a look at it after this PR)? Although adding components to the native plotter by `plotter.add` can completely handle the canvas state, there are some magicclass-way to keep API more consistent.

Reply:
I tried that! It *almost* works but I think there must be some issues in the kwargs passing as I get a wrong output cloud made of one point only..
About having something like `ui.canvas.add_points()` or not, it depends on the `magic-class` design and purpose not on `vedo`, yet what I would suggest is _not_ to have it: let vedo do it's job and rather focus on having a smooth integration with the magic-class machinery. That will save you from maintaining a possibly quite large nr of methods like `add_points()`, `add_spheres()`, `add_lines()`...
Just my 2 cents! :)
I'm thinking of adding a similar example in the vedo webpage.